py3status/modules/wifi.py

```diff
@@ -184,17 +214,17 @@ def wifi(self):
                 signal_dbm = '? dBm'
                 signal_percent = '?%'
 
-            full_text = self.py3.safe_format(
-                self.format_up,
-                dict(
-                    bitrate=bitrate,
-                    signal_dbm=signal_dbm,
-                    signal_percent=signal_percent,
-                    ip=ip,
-                    device=self.device,
-                    icon=icon,
-                    ssid=ssid,
-                ))
+        full_text = self.py3.safe_format(
+            self.format,
+            dict(
+                bitrate=bitrate,
+                device=self.device,
+                icon=icon,
+                ip=ip,
+                signal_dbm=signal_dbm,
+                signal_percent=signal_percent,
+                ssid=ssid
```

Comment:
here you should add a comma after ssid, as it is the pythonic way.  It also makes it cleaner and less error prone to add remove parameters.